draco/core/task.py

```diff
@@ -390,7 +393,13 @@ def _save_output(self, output):
             outfile = os.path.expandvars(outfile)
 
             self.log.debug("Writing output %s to disk.", outfile)
-            self.write_output(outfile, output)
+            self.write_output(
```

Comment:
I don't understand why compression parameters are now held at the `memh5.MemGroup` level. Is it just so that they can be specified in a pipeline config `save` block? Maybe there is a good reason for this to do with Zarr, but it strikes me as inconsistent with the way the compression settings are implemented in `draco` containers and the previous version of `memh5`, where compression is set for every dataset individually. I'll have to take a closer look at the `caput` PR, but in any case it strikes me at least as redundant and maybe conflicting to be specifying compression in multiple places.

Reply:
Ah, yes. I just made a similar comment above. 100% agree that we should think about how we want to be able to specify these parameters (and if we should at all?).